open-sphere-plugins/extra-icons/src/main/java/io/opensphere/milstdi/mantle/controller/StandardIcons.java

```diff
@@ -0,0 +1,217 @@
+package io.opensphere.milstdi.mantle.controller;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.net.URL;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.Iterator;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.Map;
+import java.util.Objects;
+import java.util.stream.Collectors;
+
+import org.apache.log4j.Logger;
+
+import gnu.trove.list.TIntList;
+import gnu.trove.list.array.TIntArrayList;
+import io.opensphere.core.util.collections.New;
+import io.opensphere.core.util.lang.StringUtilities;
+import io.opensphere.mantle.MantleToolbox;
+import io.opensphere.mantle.icon.IconProvider;
+import io.opensphere.mantle.icon.IconRecord;
+import io.opensphere.mantle.icon.impl.IconProviderFactory;
+
+/** Way to access MilStd-2525c icons. */
+public final class StandardIcons
```

Comment:
I would suggest using "MilStandard" or something like that in this class instead of just "Standard" because Standard makes them seem like our main icon set.  "MilStandard" or whatever would probably be easier to search for in the IDE.